core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -90,12 +96,14 @@ class HttpGoogleServicesDAO(
   val directoryScopes = Seq(DirectoryScopes.ADMIN_DIRECTORY_GROUP)
   val genomicsScopes = Seq(GenomicsScopes.GENOMICS) // google requires GENOMICS, not just GENOMICS_READONLY, even though we're only doing reads
   val billingScopes = Seq("https://www.googleapis.com/auth/cloud-billing")
+  val googleApiUri = Uri.unsafeFromString("https://www.googleapis.com")
 
   val httpTransport = GoogleNetHttpTransport.newTrustedTransport
   val jsonFactory = JacksonFactory.getDefaultInstance
   val tokenClientSecrets: GoogleClientSecrets = GoogleClientSecrets.load(jsonFactory, new StringReader(tokenClientSecretsJson))
   val tokenBucketName = "tokens-" + clientSecrets.getDetails.getClientId.stripSuffix(".apps.googleusercontent.com")
-  val cromwellMetadataBucketName = GcsBucketName("cromwell-meta-" + clientSecrets.getDetails.getClientId.stripSuffix(".apps.googleusercontent.com"))
+  val cromwellMetadataBucketName = GcsBucketName("cromwell-metadata-" + clientSecrets.getDetails.getClientId.stripSuffix(".apps.googleusercontent.com"))
+  val cromwellMetadataTopicName = ProjectTopicName.of(serviceProject, "cromwell-metadata-topic") //TODO what's good suffix for topic name?
```

Comment:
pull the topic name from config

Reply:
I was hesitating to make this in config because this change is going to be reverted at some point

Reply:
It may be reverted at some point but I bet that is months away. I think both the topic name and bucket name should be in rawls.conf like https://github.com/broadinstitute/firecloud-develop/blob/fade9286ff0aec8449121ed201ebc44c8a4d57dd/base-configs/rawls/rawls.conf.ctmpl#L51